modules/conversion-center/classes/render/core-render.php

```diff
@@ -0,0 +1,290 @@
+<?php
+
+namespace Elementor\Modules\ConversionCenter\Classes\Render;
+
+use Elementor\Modules\ConversionCenter\Widgets\Link_In_Bio;
+use Elementor\Utils;
+
+/**
+ * Class Core_Render.
+ *
+ * This class handles the rendering of the Link In Bio widget for the core version.
+ *
+ * @since 3.23.0
+ */
+class Core_Render extends Render_Base {
+
+	public function render( Link_In_Bio $widget ): void {
+		?>
+		<div class="e-link-in-bio">
+			<div class="e-link-in-bio__content">
+
+				<?php $this->render_identity_image( $widget ); ?>
+
+				<?php $this->render_bio( $widget ); ?>
+
+				<?php $this->render_icons( $widget ); ?>
+
+				<?php $this->render_ctas( $widget ); ?>
+
+			</div>
+			<div class="e-link-in-bio__bg">
+				<div class="e-link-in-bio__bg-overlay"></div>
+			</div>
+		</div>
+		<?php
+	}
+
+	private function render_ctas( Link_In_Bio $widget ): void {
+		$settings               = $widget->get_settings_for_display();
+		$ctas_props_corners     = $settings['cta_links_corners'] ?? 'rounded';
+		$ctas_props_show_border = $settings['cta_links_show_border'] ?? false;
+		$ctas_props_type        = $settings['cta_links_type'] ?? 'button';
+		$ctas_value             = $settings['cta_link'] ?? [];
+
+		$has_ctas = ! empty( $ctas_value );
+		if ( ! $has_ctas ) {
+			return;
+		}
+
+		$widget->add_render_attribute( "ctas", [
```

Comment:
```suggestion
		$widget->add_render_attribute( 'ctas', [
```
Please use single quotes everywhere

Reply:
@louiswol94 nice spot! Done

Reply:
Ok changed everywhere